INSTALL.markdown

```diff
@@ -127,15 +127,15 @@ DXX-Rebirth can be built from the Terminal (via SCons) without Xcode; to build u
 
 ##### [Homebrew](https://github.com/Homebrew/homebrew/)
 * **brew install
- gcc5
```

Comment:
Removing gcc5 is correct, since the minimum is now gcc-7.  Does Homebrew guarantee a new enough gcc that it is appropriate not to mention `gcc` at all in this list?

Reply:
Since Homebrew requires the Xcode command-line tools to be installed in order to run properly, installing packages from Homebrew guarantees a recent version of clang.  I don't have the any gcc packages installed at all (just relying on the clang suite), and it builds just fine on my machine.  That said, if gcc were needed for some reason, the gcc package in Homebrew currently installs 10.2.0, and is generally kept pretty up to date.